From d133a1c50dba2513fcc3802af96ce7f1b79db3c6 Mon Sep 17 00:00:00 2001
From: Jarek Polok <jpolok@xenophile.cern.ch>
Date: Thu, 13 Nov 2014 13:11:57 +0100
Subject: [PATCH] Define envirnment size spacious enough to hold large number
 of attributes.

---
 auth_mellon.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/auth_mellon.h b/auth_mellon.h
index e915cbfbdd33072637780145ce5d7fcf7d9ebc88..ce35959a81d470bbe22d0c71e0e710f0f0e7ff54 100644
--- a/auth_mellon.h
+++ b/auth_mellon.h
@@ -73,7 +73,7 @@
 /* Size definitions for the session cache.
  */
 #define AM_CACHE_KEYSIZE 120
-#define AM_CACHE_ENVSIZE 128
+#define AM_CACHE_ENVSIZE 2048
 #define AM_CACHE_USERSIZE 512
 #define AM_CACHE_DEFAULT_ENTRY_SIZE 196608
 #define AM_CACHE_MIN_ENTRY_SIZE 65536
-- 
2.1.0

