commit 4c924d9062406762c44ce95bdee14dadf4bfeed2
Author: Thijs Kinkhorst <thijs@kinkhorst.com>
Date:   Sun Jul 9 14:42:29 2017 +0000

    Fix some log message typos

diff --git a/auth_mellon_cache.c b/auth_mellon_cache.c
index 9a5bb0e..cd3b9b0 100644
--- a/auth_mellon_cache.c
+++ b/auth_mellon_cache.c
@@ -740,7 +740,7 @@ int am_cache_set_lasso_state(am_cache_entry_t *session,
                                          lasso_identity);
     if (status != 0) {
         ap_log_error(APLOG_MARK, APLOG_ERR, 0, NULL,
-                     "Lasso identity is to big for storage. Size of lasso"
+                     "Lasso identity is too big for storage. Size of lasso"
                      " identity is %" APR_SIZE_T_FMT ".",
                      (apr_size_t)strlen(lasso_identity));
         return HTTP_INTERNAL_SERVER_ERROR;
@@ -751,7 +751,7 @@ int am_cache_set_lasso_state(am_cache_entry_t *session,
                                          lasso_session);
     if (status != 0) {
         ap_log_error(APLOG_MARK, APLOG_ERR, 0, NULL,
-                     "Lasso session is to big for storage. Size of lasso"
+                     "Lasso session is too big for storage. Size of lasso"
                      " session is %" APR_SIZE_T_FMT ".",
                      (apr_size_t)strlen(lasso_session));
         return HTTP_INTERNAL_SERVER_ERROR;
@@ -762,8 +762,8 @@ int am_cache_set_lasso_state(am_cache_entry_t *session,
                                          lasso_saml_response);
     if (status != 0) {
         ap_log_error(APLOG_MARK, APLOG_ERR, 0, NULL,
-                     "Lasso SAML response is to big for storage. Size of "
-                     "lasso SAML Reponse is %" APR_SIZE_T_FMT ".",
+                     "Lasso SAML response is too big for storage. Size of "
+                     "lasso SAML Response is %" APR_SIZE_T_FMT ".",
                      (apr_size_t)strlen(lasso_saml_response));
         return HTTP_INTERNAL_SERVER_ERROR;
     }
diff --git a/auth_mellon_util.c b/auth_mellon_util.c
index 34cf88d..19bfb41 100644
--- a/auth_mellon_util.c
+++ b/auth_mellon_util.c
@@ -1823,7 +1823,7 @@ tokenize(apr_pool_t *pool, const char *str, bool ignore_whitespace,
             }
             if (*p != '\"') {
                 *error = apr_psprintf(pool,
-                                      "unterminated string begining at "
+                                      "unterminated string beginning at "
                                       "position %" APR_SIZE_T_FMT " in \"%s\"",
                                       start-str, str);
                 break;
